lms/djangoapps/instructor/access.py

```diff
@@ -75,6 +80,9 @@ def _change_access(course, user, level, action):
         raise ValueError("unrecognized level '{}'".format(level))
 
     if action == 'allow':
+        if level == 'ccx_coach':
+            email_params = get_email_params(course, True)
+            before, after, enrollment_obj = enroll_email(course_id=course.id, student_email=user.email, auto_enroll=True, email_students=True, email_params=email_params)
```

Comment:
this is going to be a pylint error because you've got unused variables. See https://github.com/edx/edx-platform/wiki/Python-Guidelines and https://github.com/edx/edx-platform/blob/master/docs/en_us/internal/testing.rst#python-code-style-quality